doc/mathdoc/equ-process.rst

```diff
@@ -3,149 +3,229 @@
 Process Constraints
 ^^^^^^^^^^^^^^^^^^^
 
-**Process Capacity Rule**: The constraint process capacity rule defines the variable total process capacity :math:`\kappa_{vp}`. The variable total process capacity is defined by the constraint as the sum of the parameter process capacity installed :math:`K_{vp}` and the variable new process capacity :math:`\hat{\kappa}_{vp}`. In mathematical notation this is expressed as:
+**Process Capacity Rule**: The constraint process capacity rule defines the 
+variable total process capacity :math:`\kappa_{vp}`. The variable total process 
+capacity is defined by the constraint as the sum of the parameter process 
+capacity installed :math:`K_{vp}` and the variable new process capacity 
+:math:`\hat{\kappa}_{vp}`. In mathematical notation this is expressed as: 
 
 .. math::
 
 	\forall v\in V, p\in P\colon\ \kappa_{vp} = K_{vp} + \hat{\kappa}_{vp}
 
-In script ``urbs.py`` the constraint process capacity rule is defined and calculated by the following code fragment:
+In script ``urbs.py`` the constraint process capacity rule is defined and
```

Comment:
I made these changes in my other pull request. I did not tidy up the seperate requests properly, sorry.